app/en/home/local-deployment/install/local/page.mdx

```diff
@@ -1,206 +0,0 @@
----
-title: "Local Install"
-description: "How to install Arcade"
----
```

Comment:
This page in particular can go away almost entirely:
* With arcade-mcp, you add existing arcade tools with `uv install ...` and then make a server in the normal way, importing the tools you want
* You don't need to install our client libs
* `arcade-chat` is gone
* ... and per the rest of this PR, you don't run the engine locally any more

CC @EricGustin and @Spartee - well done!